model/src/main/java/org/projectnessie/api/params/ReferencesParams.java

```diff
@@ -48,15 +57,6 @@ public boolean isFetchAdditionalInfo() {
     return fetchAdditionalInfo;
   }
 
-  private ReferencesParams(Integer maxRecords, String pageToken, boolean fetchAdditionalInfo) {
```

Comment:
Why are we moving constructors?

Reply:
because of https://github.com/projectnessie/nessie/pull/2734#discussion_r757544953 or maybe I understood your comment wrong?

Reply:
sorry about the confusion. I meant fields > constructors > other stuff